### PR TITLE
(725) Add print button and application summary to check your answers page

### PR DESCRIPTION
**Pull request by libuk**
# Context

# Changes in this PR
* Add print button
* Add application summary
* Add "back to task list" link
* Printed page styling

### Before
Screenshot 2023-12-20 at 09 35 05

#### Print
Screenshot 2023-12-19 at 15 47 32
Screenshot 2023-12-19 at 15 48 03

### After
Screenshot 2023-12-20 at 16 16 21

#### Print
Screenshot 2023-12-19 at 15 59 02
Screenshot 2023-12-20 at 16 45 15

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Are any changes required to the e2e tests?
- [ ] If you've added a new route, have you added a new
  `auditEvent`? (see `server/routes/apply.ts` for examples)
- [ ] Are there environment variables or other infrastructure configuration which needs to be included in this release?
- [ ] Are there any data migrations required. Automatic or manual?
- [ ] Does this rely on changes being deployed to the CAS API?

## Post merge checklist

Once we've merged it will be auto-deployed to the dev environment.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui).

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

**Comment by libuk** [comment]
[CYA: summary and print pdf button](https://trello.com/c/a6BcLewR/725-cya-summary-and-print-pdf-button)

File: integration_tests/pages/page.ts [hunk 1]
```diff
@@ -121,8 +121,8 @@ export default abstract class Page {
     cy.get('a').contains(label).click()
   }
 
-  shouldShowPrintButton(): void {
-    cy.get('button').contains('Save as PDF')
+  shouldShowPrintButton(text = 'Save as PDF'): void {
```
Comment (Harriethw) [inline integration_tests/pages/page.ts:124]: probably more a question for Chanel/Alex, is there a reason why we show 'Save as PDF' on one view and 'Download as a PDF' on another?

File: assets/scss/application.scss [hunk 2]
```diff
@@ -36,8 +36,16 @@ $govuk-page-width: $moj-page-width;
...
 
+.print__show {
+  display: none;
+}
+
 @media print {
   .print__hidden {
-    visibility: hidden;
+    display: none;
+  }
```
Comment (Harriethw) [inline assets/scss/application.scss:46]: nice!

**Review by Harriethw — APPROVED** [review]
nice work!

File: integration_tests/pages/page.ts (hunk 1 above)
Comment (libuk) [reply integration_tests/pages/page.ts:124]: Yeah, it's a good point. I'll confirm before merging 👍🏾

**Pull request merged**